infrastructure/code/bundle.go

```diff
@@ -165,7 +165,12 @@ func (b *Bundle) retrieveAnalysis(ctx context.Context) ([]snyk.Issue, error) {
 				for i := range issues {
 					// We only allow the issues whose file extensions are supported by the
 					// backend.
-					if !b.isAutofixSupportedForExtension(ctx, issues[i].AffectedFilePath) {
+					supported, err := b.isAutofixSupportedForExtension(ctx, issues[i].AffectedFilePath)
+					if err != nil {
+						return []snyk.Issue{}, err
+					}
```

Comment:
Why not log the error and continue? If we can't tell whether autofix is supported for some extension, does it mean we want to discard all the results?

Reply:
No, we error only if there's an error in retrieval this information. We won't be able progress here, since it's not possible to parse anything later.